app/lib/medical-transcription-analysis-stack.ts

```diff
@@ -125,6 +125,18 @@ export class MedicalTranscriptionAnalysisStack extends cdk.Stack {
         });
 
         webAppS3Bucket.addToResourcePolicy(cloudfrontPolicyStatement);
+
+        const cloudfrontStorageBucketPolicyStatement = new iam.PolicyStatement({
```

Comment:
As discussed yesterday, if you are using the storages3bucket just for storing the output of transcribe etc, you do not need to have "websiteIndexDocument" for that. You should declare it similar to : https://github.com/awslabs/document-understanding-solution/blob/c6a552bee92dcfa5d2cdb0110f67e5e7b018a3eb/source/lib/cdk-textract-stack.ts#L112